### PR TITLE
Indicators:  css modules

### PR DESCRIPTION
**Pull request by bgptr**
In this diff, I went through the indicator components and refactored them to use CSS modules, if not already done. Moreover, `ProgressRing` component was refactored to a functional component too.

Also, it includes automated tests.
Part of #2439 and #2438

### CODE REVIEW

File: app/components/indicators/AnimatedLinearProgressFull.jsx [hunk 1]
```diff
@@ -84,51 +84,55 @@ const AnimatedLinearProgressFull = ({
...
                 <div
-                  className="linear-progress-box four"
+                  className={classNames(styles.linearProgressBox, styles.four)}
                   style={{ left: leftStartingPoint + 200 }}
                 />
               )}
               {perComplete > 0.75 && perComplete < 1 && (
                 <div
-                  className="linear-progress-box five"
+                  className={classNames(styles.linearProgressBox, styles.fives)}
```
Comment (amassarwi) [inline app/components/indicators/AnimatedLinearProgressFull.jsx:119]: fives => five

**Review by amassarwi — APPROVED** [review]
Great work @bgptr - I'd recommend splitting the huge `indicators.module.css` into separate modules each located in a new nested dir along with the `.jsx` file, what do you think ?

File: app/components/indicators/AnimatedLinearProgressFull.jsx (hunk 1 above)
Comment (bgptr) [reply app/components/indicators/AnimatedLinearProgressFull.jsx]: Good catch, thx!

**Comment by bgptr** [comment]
@amassarwi Thank you for your review. I've done the splitting. It was a great idea 👌

**Comment by alexlyp** [comment]
Needs a quick rebase.

**Comment by bgptr** [comment]
@alexlyp rebased

**Review by alexlyp — APPROVED** [review]

**Pull request merged**